concepts/index.md

```diff
@@ -4,34 +4,34 @@ layout: documentation
 
```

Comment:
Note that this page was also updated by https://github.com/eclipse/smarthome/pull/1573 which was merged today - it does not speak about ESH anymore.